Lab 3 Real World Projects
Execution Logs
Student: John Harvey Calacal

In [22]:
!python main.py

CONTROL_NUM: 6
FAVORITE ARTIST: CUP OF JOE
Authorization Started
Authorization Completed
Access Level: 28
Decision: ACCESS DENIED

Signal Shutdown Simulation
Current power: 16
Current power: 15
Current power: 14
Current power: 13
Current power: 12
Current power: 11
Current power: 10
Current power: 9
Current power: 8
Current power: 7
Current power: 6
Current power: 5
Current power: 4
Current power: 3
Current power: 2
Current power: 1
System safely powered down.
Total Recursive Calls: 17

Streaming Play Counts
Generated: 1
Generated: 4
Generated: 9
Generated: 16
Generated: 25
Generated: 36
Generated: 49
Generated: 64
Generated: 81
Generated: 100
Generated: 121
Generated: 144
Generated: 169
Generated: 196
Generated: 225
Generated: 256
Total Plays: 1496
Records Processed: 16


Exercise 1 — Secure Access Authorization

CONTROL_NUM: 6
FAVORITE ARTIST: CUP OF JOE
Authorization Started
Authorization Completed
Access Level: 28
Decision: ACCESS DENIED

Exercise 2 — Recursive Signal Shutdown

Signal Shutdown Simulation
Current power: 16
Current power: 15
Current power: 14
Current power: 13
Current power: 12
Current power: 11
Current power: 10
Current power: 9
Current power: 8
Current power: 7
Current power: 6
Current power: 5
Current power: 4
Current power: 3
Current power: 2
Current power: 1
System safely powered down.
Total Recursive Calls: 17

Exercise 3 — Streaming Play Counts

Streaming Play Counts
Generated: 1
Generated: 4
Generated: 9
Generated: 16
Generated: 25
Generated: 36
Generated: 49
Generated: 64
Generated: 81
Generated: 100
Generated: 121
Generated: 144
Generated: 169
Generated: 196
Generated: 225
Generated: 256
Total Plays: 1496
Records Processed: 16